__tests__/app.test.js

```diff
@@ -85,4 +86,37 @@ describe('4 get/api/articles/:article_id', () => {
   test('when inputted a different data type, returns status 400 ', () => {
     return request(app).get("/api/articles/nonsensicalgibberish").expect(400)
     })
-  })
\ No newline at end of file
+  })
+
+
+  describe('5 get/api/articles/id/comments', () => {
+    test('returns 200, with the comments related to the article id', () => {
+        return request(app).get("/api/articles/5/comments").expect(200)
+        .then((res)=>{
+            expect(res.body.comments.length).toBe(2)
+            res.body.comments.forEach((comment)=>{
+            
+               expect(comment).toEqual(expect.objectContaining({
+                comment_id: expect.any(Number),
+                article_id : 5,
+                body : expect.any(String),
+                author: expect.any(String),
+                votes: expect.any(Number),
+                created_at : expect.any(String)
+
+               }))
+               
+            })
+            expect(res.body.comments).toBeSortedBy("created_at",{
+                descending:true,
+                coerce: true
+            })
+        })
+    })
+    test('tests for a valid article, but no comments are found, so should return an empty array', () => {
+        return request(app).get("/api/articles/4/comments").expect(200).then((res)=>{
+            expect(res.body.comments.length).toBe(0)
+            expect(res.body.comments).toEqual([])
+        })
+    });
+  });
```

Comment:
Needs a couple extra tests for: 
- an invalid id (e.g /api/articles/banana/comments)
- an id of a non - existant endpoint